src/main/java/uk/gov/justice/hmpps/prison/repository/sql/ScheduleRepositorySql.kt

```diff
@@ -194,13 +194,13 @@ enum class ScheduleRepositorySql(val sql: String) {
         OIS.COMMENT_TEXT AS "COMMENT",
         OIS.TO_INTERNAL_LOCATION_ID AS LOCATION_ID
         FROM OFFENDER_IND_SCHEDULES OIS
-        INNER JOIN OFFENDER_BOOKINGS OB ON OB.OFFENDER_BOOK_ID = OIS.OFFENDER_BOOK_ID AND OB.ACTIVE_FLAG = 'Y'
+        INNER JOIN OFFENDER_BOOKINGS OB ON OB.OFFENDER_BOOK_ID = OIS.OFFENDER_BOOK_ID AND OB.ACTIVE_FLAG = 'Y' AND OIS.AGY_LOC_ID = OB.AGY_LOC_ID
         INNER JOIN OFFENDERS O ON OB.OFFENDER_ID = O.OFFENDER_ID
```

Comment:
This change is needed to avoid prisoner appointments showing on schedules at the wrong prison when the person has been transferred.

Reply:
It is a great question. Does the booking AGY_LOC_ID alter when someone is on TAP? I've got some data requests to do later this morning so will have a look at this in preprod.